src/winrtble/peripheral.rs

```diff
@@ -263,6 +269,7 @@ impl Peripheral {
                 BluetoothAddressType::Random => Some(AddressType::Random),
                 _ => None,
             };
+            drop(address_type_guard);
```

Comment:
Again, this is just about to go out of scope and be dropped anyway.

Reply:
Ok, I was not sure about that.. I discard all of them.

Reply:
I just commit the update